tests/foreman/ui/test_discoveredhost.py

```diff
@@ -598,10 +673,28 @@ def test_positive_update_default_org(self):
 
         @Assert: Default org should be successfully changed for multiple hosts
 
-        @caseautomation: notautomated
-
         @CaseLevel: System
         """
+        new_org = gen_string('alpha')
+        entities.Organization(name=new_org).create()
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            with LibvirtGuest() as pxe_1_host:
+                host_1_name = pxe_1_host.guest_name
+                self._assertdiscoveredhost(host_1_name)
+                with LibvirtGuest() as pxe_2_host:
+                    host_2_name = pxe_2_host.guest_name
+                    self._assertdiscoveredhost(host_2_name)
+                    hostnames = [host_1_name, host_2_name]
+                    for hostname in hostnames:
+                        host = self.discoveredhosts.search(hostname)
+                        if not host:
+                            raise UIError(
+                                'Could not find the selected discovered host '
+                                '"{0}"'.format(hostname)
+                            )
+                        self.discoveredhosts.navigate_to_entity()
```

Comment:
Isn't this line redundant? `self.discoveredhosts.search()` will navigate to entity anyway